scripts/dedicated-setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+
+
+PASSWORD=$(openssl rand -base64 12)
```

Comment:
Can this be something simple? (good old Password1)

Reply:
I'd prefer to not use that. The script outputs the password